StackExchangeRedis/NHibernate.Caches.StackExchangeRedis.Tests/CacheRegionStrategyFactory.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using NHibernate.Caches.StackExchangeRedis.Tests.MemoryCaches;
+using StackExchange.Redis;
+
+namespace NHibernate.Caches.StackExchangeRedis.Tests
+{
+	public class CacheRegionStrategyFactory : DefaultCacheRegionStrategyFactory
```

Comment:
Was there a reason not to add support for these new strategies directly in the `DefaultCacheRegionStrategyFactory` @maca88 ?

See #123

Reply:
Looking to include it in the default factory, I have now seen the reason, see #123.